.github/workflows/ci.yml

```diff
@@ -142,12 +147,22 @@ jobs:
         with:
           path: app
 
+      - name: Get workflow uses data
+        id: get_ref
+        run: |
+          workflow_yaml=$(cat app/.github/workflows/${{ github.workflow }}.yml)
+          uses_data=$(echo "$workflow_yaml" | grep 'uses:')
+          tag=$(echo $uses_data | sed -n 's/.*@\(.*\)/\1/p')
+          echo "##[set-output name=ci_version;]$(echo $tag)"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Same comments as above.

+ @Patel-world - Can you verify if this approach is 100% working? My bet is it's not. This might be working for analyze, but can make sure this is working out for other steps via manually checking the workflow run in boilerplate?

Reply:
Hint - See how `image_ref` output is defined and is used.